components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,6 +195,35 @@
","wife of William Barton","Margaret Patoune","Margaret Laurymer","Janet Tucidie","Janet Allane","Janet Bowis","Effie Pothif","wife of Robert Wir","Janet Coldane","Janet Robison","Elspeth Chib","Bessie Nicoson","Margaret Waldon","Marin Coran","Agnes Williamsone","Helen Reid","Patrick Meikle","Margaret Blak","John Homme","Janet Pook","Issobell Cathie","Susanna Bannytyne","Christian Blaik","Jonnet Kempe","Anna Pilmure","Marion Blaik","Jeane Martin","Jean Kerse","Janet Symson","Bessie Lamb","Jon Russell","Jon Lyll","wife of Thomas Hay","Marion Cumming","George Binnie","George Lacost","Helen Nicolson","David Schankis","Neving","Thomas Weir","Cristian Brake","James Anderson","Janet Anderson","Jeane Laying","James Beverley","Grissel McIlney","Bessie Paine","Margaret McGuffock","Grissell Rae","Jonet Howat","Janet Robertson","Jonet Losk","Hopkin","Janet Fisher","Helen Wilson","Joan Graham","Marion McRae","Issabell Reid","Catharin Lorimer","Jean Gray","Jennet Broun","John Black","Margaret Wilkin","John Glass","John Paterson","Jean Woodrow","Elspeth Rule","Christian Bonn","Geddoch McAlester Boyne","Bornlie","Gilbert Mowat","Patrick Barnett","Bessie Andro","Agnes Gune","Jonet Dow","Marshall Sutherland","Adam Barnie","Marione Kenock","John Harper","William McAngus","Thomas Mrgillie Lukell","Margaret Nakoch","Gretchach Unchach","Henrie Lyell","Donald McChereich","Jean Rutherd","Elspeth Chousley","Christian Wilkieson","Katie","Margaret Hay","Isobel Insh","Johne Young","Charles Wat","Janet Campbell","Margaret Nidrie","Margaret Dalgleish","Margaret Garvie","Barbara Honeyman","Jonet Hood","James Og","Elspet Strachund","Christiane Arcly","George Milton","Agnes Nevie","Margrat NcWilliam","wife of Wilsone","Katherine Gilmore","Issobell Pain","Marion Wood","James Jervie","Jonet Mackley","Agnes Baxter","wife of Alexander Aitkin","Mathew Connell","Mareoun","Scota Bess","Margaret Wick","Eupham Wright","Adam Donaldson","Jonet Durie","Elspeth Durie","Agnes Anderson","Issobell Davidson","Bessie Dock","Euphame Henderson","Agnes Mertene","Bessie Smith","Marie Dick","Margaret Gray","Margaret Lummisdane","Issobell Chrystie","Marie Broun","Helen Wood","Jonet Inglyses","Jonet Lummisdane","Issobell Oster","Beatrix Dolt","Bessie Boyd","Jonet Kennedy","wife of George Mot","Jonet Young","Bessie Bell","Janet M'Allexander","Effie McBrwme","Helen Pennant","Agnis Quhytelaw","Elizabeth Hunter","Agnes Peramorris","Geilis Feirour","Jonet Barroman","Agnes Gordoun","Bessie Ramsay","Bessie Lamb","Gelis Durye","Bessye Brodye","Maldye Sturrok","Issobell Robertsoun","Jonet Gardyn","Dame Logye","Johnne Stevin","Henrye Stevin","wife of Johnne Durwart","Jonet Harbertson","Jonet Newton","Cristiane Twych","Betie Wobster","Jonet Duncane","Katherine Cusing","Issobell Sutye","Law","Fischearis Docter","James Ochterlonyes","Margaret Mowbray","Dowglass","Cristeane Incheot","Lyvelyke","James Kilgour","Effie Roger","Cristeane Johnnestoun","Jonet Curchan","James Chalmer","Catherine Campbell","Cristeane Jak"];
```

Comment:
Is there any other way to identify these individuals other than their name? Is there a rule we can apply to the date that is passed in? Perhaps not, just trying to understand the context of the issue.

Reply:
I tried to think of other ways like the date beginning with 01-01-YYYY, but this would exclude dates that shouldn't be excluded for example if the actual date was the first of January and not just that only the full date wasn't available. I do agree this doesn't seem more efficient way to do this but was struggling to think of other ways. This is quite hard to explain so not sure I did a great job.

Reply:
These names are no longer needed. Ruby has removed them as we instead are using date precision provided by Wikipedia to identify when a translation is needed